lua/nvim-treesitter/utils.lua

```diff
@@ -6,7 +6,7 @@ local M = {}
 
 function M.setup_commands(mod, commands)
   for command_name, def in pairs(commands) do
-    local call_fn = string.format("lua require'nvim-treesitter.%s'.commands.%s.run(<f-args>)", mod, command_name)
+    local call_fn = string.format("lua require'nvim-treesitter.%s'.commands.%s.run('<bang>')(<f-args>)", mod, command_name)
```

Comment:
Wow, that's a bit of a breaking change when we would do that as also other nvim-treesitter modules depend on that. Wouldn't `.commands['%s<bang>']` work also? Then, modules could just opt-in to by providing a `['install!']` function.